### DISCUSSION TITLE
Is there a way to crop a high quality video without lowering the quality?

### DISCUSSION THREAD

**Discussion by stephane-archer** [mifi/lossless-cut#2042] [Q&A] [answered]
if I crop a 9:16 image (vertical) from a 1920x1080 video I should get a 608x1080 image.

How can I get the 608x1080 video without losing quality? Can Lossless-Cut can do that? If not do you know any alternative?

**Comment by mifi** [comment, marked as answer]
lossless cropping is not possible, see faq https://github.com/mifi/lossless-cut/blob/master/issues.md#faq
you can use other tools like openshot, shotcut etc in order to to crop videos

**Reply by stephane-archer** [reply]
Lossless cropping without transcoding is not possible but is possible if you output to a lossless codec. Is this correct ? 

Also, can't we crop by discarding "Macroblocks" in the same way we can trim on key frames ?

**Reply by mifi** [reply]
correct. lossless codec would work but it requires a slow encode and is out of the scope of losslesscut. there's some discussion around lossless cropping here but not yet supported: #643

**Reply by ani917** [reply]
> shotcut

I didn't find cropping in this program, but there are resolution changes

**Comment by ubershmekel** [comment]
Seems cropping is now available based on the main readme...

**Reply by mifi** [reply]
yes, it's inside the tracks panel, click the menu on the video track that you want to crop

**Reply by ewb-git** [reply]
The implementation is awful...there should be some button in the main window that then lets you crop the video iphone/photoshop style with draggable borders. Well...maybe I'll try implement it with claude.
